sql.txt

```diff
@@ -1,14 +1,38 @@
 (1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
 Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
 
+SELECT CustomerName, ContactName, Country FROM customers ORDER BY country ASC LIMIT 10 OFFSET 10;
+
+SELECT customerid, CustomerName, ContactName, Country FROM customers ORDER BY country ASC LIMIT 10 OFFSET 10;
+
+
 (2) Select all columns from the Customer table where the ContactName starts with A.  Paste your SQL statement below.
 
+SELECT * FROM Customers where CustomerName like "A%";
+
+
 (3) Select all columns from the OrderDetails table where the ProductID is 51 and the quantity is greater than 10.
 Paste your SQL statement below.
 
+SELECT * FROM OrderDetails WHERE productID = 51 AND quantity > 10;
+
+
 (4) Insert 3 rows into the Products table.  Note that you will have to specify a valid SupplierID and CategoryID, 
 corresponding to rows from the Supplier and Category tables.  Paste your three SQL statements below.
 
+INSERT INTO products VALUES 
+(NULL, "sams", 8, 4, "10 boxes", 50)
+(NULL, "moonlight", 15, 8, "100 boxes", 750),
+(NULL, "summer swish", 20, 2, "45 boxes", 800);
+
+
 (5) Update the two top rows of the Products Table to increase the price by 1.50. (Get SQL to do the addition for you.)  Paste your SQL statement below.
 
+
+UPDATE products SET price = price + 1.50 WHERE productid BETWEEN 1 AND 2; 
+
+
 (6) Delete all rows of the Products Table where the price is less than 7.00. Paste your SQL statement below.
+
+
+DELETE FROM products WHERE price < 7.00;
```

Comment:
Looks good to me.  Please, next time make sure that you submit you work from your repo.